### ISSUE TITLE
Add "View Comment" context action to channels

### ISSUE THREAD

**Issue by dexgs** [mumble-voip/mumble#6525]
### Context

Currently, the only way to view channel comments is to hover the cursor over the channel, or click the comment indicator. Either way, the channel comment is only viewable as a pop-up, not a proper window which allows scrolling/copy-pasting.

Currently, there is no way to copy text from a channel comment or scroll through a channel comment that is too big to fit on screen at once without having "write ACL" permission on the channel and using the comment editor.

### Description

There is currently a "View Comment" context action on users which lets you view their comment in a window which allows text selection and scrolling. This "View Comment" action should also be available on channels.

### Mumble component

Client

### OS-specific?

No

**Comment by Ali-Hon** [comment]
Hi, I'm interested in contributing!

**Comment by Krzmbrzl** [comment]
@Ali-Hon thanks for your interest. However, this particular issue has already been addressed by @dexgs in the linked pull request (we just did not get around reviewing it).

**Comment by Ali-Hon** [comment]
Oki :) I will keep looking for one, defect or feature but feel free to recommend one or point me towards an important one that has not been picked up yet :)

**Comment by Aryanonit** [comment]
is this issue still open or solved?

**Comment by dexgs** [comment]
> is this issue still open or solved?

It's still open and unsolved. My PR #6526 is currently a draft because it still requires some work. I haven't made the time to go back to it but I will finish it eventually unless someone else beats me to it.

**Comment by ccoppinrulez229** [comment]
Is this issue still unresolved? I'm willing to give it a shot.

**Comment by Krzmbrzl** [comment]
Yes, #6526 is still in draft mode. From what dexgs said, I assume that they are fine with someone als finalizing the PR (keeping proper (co-)authorship, of course)

**Comment by ccoppinrulez229** [comment]
Good to know, thanks

**Comment by ccoppinrulez229** [comment]
This is my first contribution ever and I'm in part doing it for a school project. Could anyone tell me which files (and where) I can specifically find the logic in the code for right clicking to open the context menu on channels and users respectively, the context menu options themselves, as well as tell me which ui file will be called for opening the comment window for users?

Sorry to bombard. It's my first time tackling a project of this scope and I just want to get started.

**Comment by Hartmnt** [comment]
@ccoppinrulez229 If you take a look at the existing pull request #6526, you can select individual commits.

Check this out commit: https://github.com/mumble-voip/mumble/pull/6526/commits/ef5d09238354c10c90cc49016ae2be90bdc1072c
You can see the individual files that were changed as well as which parts of the code. Those are the code changes that need some improvement, as it almost solves the problem discussed in this issue, but not quite. That's why we (the maintainers of Mumble) have added some comments to that pull request.

If you want to tackle this particular issue, spend your time wisely and build on top of the work that the OP of the pull request has already completed. You can even check out his repository/branch and compile that to get an idea what is working and what is not working yet.

**Comment by ccoppinrulez229** [comment]
I see, thank you.

**Comment by ccoppinrulez229** [comment]
Thanks for letting me know. Knowing exactly where to look helped a lot. I was able to create my own solution that seems to be working effectively so far. I will keep further testing.
